NetworkingDemo/NetworkingDemoLauncher.swift

```diff
@@ -12,47 +12,25 @@ import AlamofireNetworkActivityIndicator
 import AlamofireNetworkActivityLogger
 
 class NetworkingDemoLauncher {
+    // Provide a valid session token for the demo app to work properly.
+    static let sessionToken = ""
```

Comment:
if you need to find out how to get a valid token to be able to run the demo probably nobody is going to run it. It would be nice if we could change the demo to work with the new backend that gabi made, that doesn't require any authentication.

Reply:
Ah yes! I thought about that and then forgot. I'll talk with gaby